api/WheelEvent.json

```diff
@@ -320,10 +320,10 @@
               "version_added": null
             },
             "safari": {
-              "version_added": true
+              "version_added": null
```

Comment:
It looks like based upon your testing, this should be `false`?

Reply:
indeed !